spring-cloud-dataflow-composed-task-runner/pom.xml

```diff
@@ -53,6 +53,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-batch</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Would adding this allow to drop `spring-cloud-dataflow-server-core` as a test dependency? I'd really like to get this app to get build before server core.

Reply:
Removed.